lib/driver.js

```diff
@@ -332,20 +335,18 @@ class XCUITestDriver extends BaseDriver {
       await this.stopRemote();
     }
 
-    await this.runReset();
+    if (this.opts.resetOnSessionStartOnly === false) {
```

Comment:
This is just to make sure _this.opts.resetOnSessionStartOnly_ is not set to undefined or whatever. I find this approach safer.